openslides/users/static/js/users/site.js

```diff
@@ -686,10 +688,27 @@ angular.module('OpenSlidesApp.users.site', [
 
         // Export as PDF
         $scope.pdfExportUserList = function () {
-            var filename = gettextCatalog.getString("List of participants")+".pdf";
-            var userListContentProvider = UserListContentProvider.createInstance($scope.usersFiltered, $scope.groups);
-            var documentProvider = PdfMakeDocumentProvider.createInstance(userListContentProvider);
-            PdfCreate.download(documentProvider.getDocument(), filename);
+            var imageMap = {};
+            var imagePromises = [];
+            var imageSources = [];
+
+            imageSources.push(Config.get('logo_pdf_header').value.path);
+            imageSources.push(Config.get('logo_pdf_footer').value.path);
+
+            imagePromises = _.map(imageSources, function (image_source) {
```

Comment:
Can you move this into a more generic place (Because of duplicate in motion/pdf.js)? I think, now with logos in every PDF, we always need to have base64-imagedata. My proposal:
Pass an array of image sources (`['/path/to/img.jpg', 'pic.jpg']`) to the content provider. This is needed for motions; for users an empty array.
Append the two image sources in the content provider for header and footer to the array and convert it into base64. This may be done in some kind of `BaseContentProvider`, so we keep DRY in every content provider.
In the documentProvider use the base64Strings from the given contentProvider.